Jenkinsfile

```diff
@@ -57,7 +57,9 @@ pipeline {
                             if(!infra.isTrusted()) {                                
                                 deleteDir()
                                 checkout scm
-                                sh "make build ; docker system prune --force --all"
+                                sh 'make build'
```

Comment:
I'm not trying to pedantic, but you could combine this into a single `sh` call to make it a tiny bit more efficient:

```
sh '''
make build
make test
docker system prune --force --al
```

Reply:
Yeah I know. Does it exit if one of the commands fails or should I set -e?

Reply:
Yes, it should. Though we want the prune to always run if possible.